packages/openapi-react-query/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "openapi-react-query",
   "description": "Fast, type-safe @tanstack/react-query client to work with your OpenAPI schema.",
-  "version": "0.1.3",
+  "version": "0.1.4",
```

Comment:
cc @kerwanp This could arguably be a minor since it’s a “breaking change” but we haven’t really committed to that with this package yet. But feel free to publish a minor bump if you feel we should / indicate this is a breaking change.

This particular change is probably something that won’t affect most people, and is arguably a bugfix, but I just erred on the side of caution with `openapi-fetch`.